### DISCUSSION TITLE
for input string "1024"

### DISCUSSION THREAD

**Discussion by bq2015** [MetaCubeX/ClashMetaForAndroid#253] [Q&A]
订阅保存时，报这个错，是什么意思。有谁知道。clashForAndroid是可以，clashforwidows也可以。就clashmeta报错。

**Comment by bq2015** [comment]
clashmeta  流量字段，字符转换int类型前，没做去空判空处理

**Reply by qingmuhy744** [reply]
这么久了，这个问题好像还是存在。
